packages/amazonq/src/lsp/client.ts

```diff
@@ -134,7 +149,27 @@ export async function startLanguageServer(
             activate(client, encryptionKey, resourcePaths.ui)
         }
 
-        const refreshInterval = auth.startTokenRefreshInterval()
+        const refreshInterval = auth.startTokenRefreshInterval(10 * oneSecond)
+
+        const sendProfileToLsp = async () => {
+            try {
+                const result = await client.sendRequest(updateConfigurationRequestType.method, {
+                    section: 'aws.q',
+                    settings: {
+                        profileArn: AuthUtil.instance.regionProfileManager.activeRegionProfile?.arn,
+                    },
+                })
+                client.info(
+                    `Client: Updated Amazon Q Profile ${AuthUtil.instance.regionProfileManager.activeRegionProfile?.arn} to Amazon Q LSP`,
+                    result
+                )
+            } catch (err) {
+                client.error('Error when setting Q Developer Profile to Amazon Q LSP', err)
```

Comment:
non-blocker/future reference: these should probably be logged in `logger`, not `client`